docs/core/actions.rst

```diff
@@ -84,15 +84,23 @@ The only thing your action server needs to install is ``rasa-sdk``:
     create a separate container for your action server. In this
     separate container, you only need to install ``rasa-sdk``.
 
-If your actions are defined in a file
-called ``actions.py``, run this command:
+If you have ``rasa`` installed, run this command to start your action server:
 
 .. code-block:: bash
 
-    python -m rasa_sdk --actions actions
+    rasa run actions
 
 .. _custom_action_example:
 
+Otherwise, if your actions are defined in a file
+called ``actions.py`` and you do not have ``rasa`` installed, run this command:
```

Comment:
Actually, if you run `rasa run actions` your actions should also be defined in `actions.py`. If the file as a different name, such as `custom_actions.py`, you need to specify it via `rasa run actions --actions custom_actions`.

Reply:
Okay, so do you think I should add that to the documentation? I just tried to follow the original and add that you can also use `rasa run actions` (which is completely missing right now); however, we need to keep the part about how to run your action server without ``rasa`` being installed, which was the main point of this section (not that your action need to be defined in ``actions.py``)

Reply:
As it is not stated anywhere else, I guess it makes sense to mention that your actions file should be called `actions.py`. But would be great if you mention it at the beginning as it applies to both commands.